### ISSUE TITLE
Light Groups in Get node

### ISSUE THREAD

**Issue by vandalon** [andreypopov/node-red-contrib-deconz#9]
** Describe your idea here**

I remember now what I wanted to say before :)

It would be nice to have the light groups in the get node so we can check if a light in a group is one before starting some kind of automation :)

**Comment by vandalon** [comment]
#21

**Comment by vandalon** [comment]
Fixed with #21
